### DISCUSSION TITLE
Pinterest Integration

### DISCUSSION THREAD

**Discussion by fuekiin** [outline/outline#7923] [Ideas]
Hello everyone,

I’ve been developing a Pinterest embed for my personal use and wanted to gauge interest in making it available for all users. I believe there could be several valuable use cases for those who use Pinterest to collect and organize visual ideas. 

Some Ideas:

**Office Moodboards:** Create visual boards for workspace design ideas, improving office layouts, or enhancing team spaces.

**Marketing Inspiration:** Gather social media post concepts, campaign visuals, or other marketing inspiration in one place.

**Team Collaboration on Design Ideas:** Allow creative teams to share mood boards for product or branding visuals.

**Event Planning:** Curate ideas for company events, team-building activities, or conferences.

I know there have been discussions before ([GitHub Issue #6074](https://github.com/outline/outline/pull/6074)). That's why I also wanted to ask if there’s a chance for it to be merged at all @tommoor?

The Embed is working for Boards, Profiles and Pins.

<img width="1680" alt="Bildschirmfoto 2024-11-11 um 17 15 01" src="https://github.com/user-attachments/assets/f72aa2bf-8d35-4c65-9e21-b9a95582244d">

<img width="1680" alt="Bildschirmfoto 2024-11-11 um 17 15 06" src="https://github.com/user-attachments/assets/3c7d525b-d155-4b22-9f3f-0d75e5dbee49">

<img width="1680" alt="Bildschirmfoto 2024-11-11 um 17 22 12" src="https://github.com/user-attachments/assets/178e5014-2062-490a-9abf-09eea3234729">

**Comment by tommoor** [comment]
Your version looks good, check out how it functions in dark mode too ;) 

Would be happy to include something like this.

**Reply by fuekiin** [reply]
Thank you @tommoor. [Here you can find the PR](https://github.com/outline/outline/pull/7930)
